Snakefile

```diff
@@ -330,33 +336,10 @@ def terminate_if_cutout_exists(config=config):
             raise Exception(
                 "An option `build_cutout` is enabled, while a cutout file '"
                 + cutout_fl
-                + "' still exists and risks to be overwritten. If this is an intended behavior, please move or delete this file and re-run the rule. Otherwise, just disable the `build_cutout` rule in the config file."
+                + "' still exists and risks to be overwritten. If this is an intended behavior, please move or delete this file and re-run the rule. Otherwise, just disable the `build_cutout` and `retrieve_cutout` rule in the config file."
             )
 
 
-if config["enable"].get("build_cutout", False):
```

Comment:
Hello!
just minor comment beyond what already discussed. Please restore this; I believe it was a refuse.
Many thanks!